predicators/utils.py

```diff
@@ -466,13 +469,13 @@ def line_segments_intersect(seg1: LineSegment, seg2: LineSegment) -> bool:
     """
 
     def _subtract(a: Tuple[float, float], b: Tuple[float, float]) \
-        -> Tuple[float, float]:
+            -> Tuple[float, float]:
```

Comment:
are these changes left over from your yapf upgrade? as long as the yapf check passes on github it's fine though